pkg/orchestrator/util.go

```diff
@@ -25,27 +29,83 @@ const (
 	K8sCluster
 )
 
+var (
+	telemetryTags = map[NodeType][]string{
+		K8sCluster:    getTelemetryTags(K8sCluster),
+		K8sDeployment: getTelemetryTags(K8sDeployment),
+		K8sNode:       getTelemetryTags(K8sNode),
+		K8sPod:        getTelemetryTags(K8sPod),
+		K8sReplicaSet: getTelemetryTags(K8sReplicaSet),
+		K8sService:    getTelemetryTags(K8sService),
+	}
+)
+
 // NodeTypes returns the current existing NodesTypes as a slice to iterate over.
 func NodeTypes() []NodeType {
-	return []NodeType{K8sCluster, K8sNode, K8sPod, K8sReplicaSet, K8sDeployment, K8sService}
+	return []NodeType{
+		K8sCluster,
+		K8sDeployment,
+		K8sNode,
+		K8sPod,
+		K8sReplicaSet,
+		K8sService,
+	}
 }
 
 func (n NodeType) String() string {
 	switch n {
-	case K8sNode:
-		return "Node"
 	case K8sCluster:
 		return "Cluster"
-	case K8sService:
-		return "Service"
-	case K8sReplicaSet:
-		return "ReplicaSet"
 	case K8sDeployment:
 		return "Deployment"
+	case K8sNode:
+		return "Node"
 	case K8sPod:
 		return "Pod"
+	case K8sReplicaSet:
+		return "ReplicaSet"
+	case K8sService:
+		return "Service"
 	default:
 		log.Errorf("Trying to convert unknown NodeType iota: %v", n)
 		return ""
 	}
 }
+
+// Orchestrator returns the orchestrator name for a node type.
+func (n NodeType) Orchestrator() string {
+	switch n {
+	case K8sCluster:
```

Comment:
nit: I had the feeling that our code base seems to prefer instead.
```
case: x,y,c:
 something
```

Any specific reason for this implementation?

Reply:
that was mainly for clarity as the list will get longer
but we can also use a more compact form until we add more resources